### PR TITLE
UIView extension to assist with positioning and sizing UIViews

### PR DESCRIPTION
**Pull request by NathanjTribe**

### CODE REVIEW

File: UIViewExtension.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		get{ return self.frame.size.width}
+		set(newValue){self.frame.size.width = newValue }
+	}
+	
+	var height: CGFloat {
+		get{ return self.frame.size.height }
+		set(newValue){ self.frame.size.height = newValue }
+	}
+	
+	var origin: CGPoint {
```
Comment (viewDidAppear) [inline UIViewExtension.swift:43]: Would this perhaps be better renamed `position` to not conflict with the existing `origin` point being returned?

**Comment by viewDidAppear** [comment]
🚀

File: UIViewExtension.swift (hunk 1 above)
Comment (NathanjTribe) [inline UIViewExtension.swift:43]: Yeah considered naming it that. There's no origin property on UIView though, so there is no conflict. I chose `origin` to keep it consistent, as that is the name of the property on the `frame` object that is being changed. What does @mark-jTribe think?

Comment (marktrobinson) [inline UIViewExtension.swift:43]: I think Ben raises a good point but since it's actually the origin, I think it's fine overall.

**Pull request merged**